src/database/migrations/z1631138691-create-notification.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+// eslint-disable-next-line no-undef
+module.exports = {
+    async up(queryInterface, Sequelize) {
+        return queryInterface.createTable('app_notification', {
+            id: {
+                type: Sequelize.INTEGER,
+                autoIncrement: true,
+                primaryKey: true,
+            },
+            address: {
+                type: Sequelize.STRING(44),
+                references: {
+                    model: 'user',
+                    key: 'address',
+                },
+                onDelete: 'CASCADE',
+                allowNull: false,
+            },
+            type: {
+                type: Sequelize.INTEGER,
+                allowNull: true,
+            },
+            params: {
+                type: Sequelize.JSON,
```

Comment:
Any reason to JSON instead of string?
I see the possibilities, but I'm wondering if it's worth the extra work :thinking:

Reply:
I thought to use JSON to be easier to pass as a parameter in some action on the application. If we saved more than one param, on the app, will be necessary some treatment, maybe making a `split` to extract from a string the params. As an object, we can use it directly.

Reply:
changed to string!